tests/postprocess_merge_duplicate_nodes_test.py

```diff
@@ -0,0 +1,106 @@
+import pytest
```

Comment:
Great tests! These are clear, explicit, and make it obvious what the purpose of the function is.